gem/Rakefile

```diff
@@ -1,2 +0,0 @@
-require "bundler/gem_tasks"
-task :default => :spec
```

Comment:
I don't think we use this Rakefile?

Removing it means we can remove the bundler/rake dependencies in our gemspec file, which would make doing a bundle install of this gem faster.

Reply:
Removed: https://github.com/puppetlabs/pupperware/pull/170